scripts/docker-integration-tests/common.sh

```diff
@@ -36,3 +36,99 @@ function retry_with_backoff {
 
   return $exitCode
 }
+
+function setup_single_m3db_node {
+  wait_for_db_init
+}
+
+function wait_for_db_init {
+  echo "Sleeping for a bit to ensure db up"
+  sleep 15 # TODO Replace sleeps with logic to determine when to proceed
+
+  echo "Adding namespace"
+  curl -vvvsSf -X POST 0.0.0.0:7201/api/v1/namespace -d '{
+    "name": "agg",
+    "options": {
+      "bootstrapEnabled": true,
+      "flushEnabled": true,
+      "writesToCommitLog": true,
+      "cleanupEnabled": true,
+      "snapshotEnabled": true,
+      "repairEnabled": false,
+      "retentionOptions": {
+        "retentionPeriodNanos": 172800000000000,
+        "blockSizeNanos": 7200000000000,
+        "bufferFutureNanos": 600000000000,
+        "bufferPastNanos": 600000000000,
```

Comment:
Nit: Probably better to use string and durations? i.e. `"bufferPast": "10m"` etc